### ISSUE TITLE
Example scripts for fine-tuning Qwen3-VL

### ISSUE THREAD

**Issue by llajan** [QwenLM/Qwen3-VL#1547]
Thanks for releasing a great set of Qwen3-VL models!
The fine-tuning examples in the repo mostly apply to Qwen 2.5. Can some examples be provided for fine-tuning Qwen3-VL?
The repo mentions that zero3 and load-balancing losses are unsupported, but further details are unavailable. Some specific examples for fine-tuning Qwen3-VL models would be helpful. 
Thanks!

**Comment by wuzhaodongaipython** [comment]
+1

**Comment by yszhli** [comment]
+1， 如果使用zero3,  程序在开始的时候就出现卡住的现象

**Comment by John-Ge** [comment]
Thank you for your feedback, we would like to add some finetuning scripts for Qwen3VL series models. Zero-3 is not supported for MOE models, so you have to use zero-2 for Qwen3VL-30B-A3B.

**Comment by xuzhang37** [comment]
> Thank you for your feedback, we would like to add some finetuning scripts for Qwen3VL series models. Zero-3 is not supported for MOE models, so you have to use zero-2 for Qwen3VL-30B-A3B.

使用zero2的时候会有显存OOM的问题（80g显存），batchsize per device已经设置为1，有什么解决的办法吗？

**Comment by John-Ge** [comment]
Use 32 GPUs or you could make up a PR using FSDP: https://huggingface.co/docs/transformers/fsdp.

**Comment by dannikay** [comment]
+1. 

my fine tuning on text+image dataset starts yielding NaN loss after 1st step. it would be nice to have a reference script to start with.
